data/test/scenarios/test_clear.cfg

```diff
@@ -1,6 +1,17 @@
-# Test that clearing multiple variables works, also that
-# the WML `_G` analogue works.
+# Test that clearing multiple variables works, also that the WML `_G` analogue works.
```

Comment:
What does "the WML _G analogue" mean?

Reply:
@CelticMinstrel can you answer this?

Reply:
I guess it must be referring to `wml.all_variables`?

Reply:
Alright, I'll add that to the comment then.